@xen-orchestra/lite/src/components/ui/UiProgressBar.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="ui-progress-bar" :class="`color-${color}`">
+    <div class="fill" />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import type { Color } from "@/types";
+import { computed } from "vue";
+
+const props = withDefaults(
+  defineProps<{
+    value: number;
+    color?: Color | "custom";
+    maxValue?: number;
+  }>(),
+  { color: "info", maxValue: 100 }
+);
+
+const progressWithUnit = computed(() => {
+  const progress = Math.round((props.value / props.maxValue) * 100);
```

Comment:
Do we need to round it?

Reply:
No, it was used in template before. I'll remove it.